examples/hitl/rearrange_v2/ui.py

```diff
@@ -480,8 +485,13 @@ def _draw_hovered_pickable(self) -> None:
         translation = managed_object.translation
         reachable = self._is_within_reach(translation)
         color = COLOR_VALID if reachable else COLOR_INVALID
-        aabb = managed_object.collision_shape_aabb
-        self._draw_aabb(aabb, managed_object.transformation, color)
+        self._gui_drawer._client_message_manager.draw_object_outline(
```

Comment:
A new method can be added to `GuiDrawer` for drawing outlines, which would be handling both local `habitat-sim` and remote outlining.

I haven't got around doing this yet.